src/dmd/backend/gflow.d

```diff
@@ -580,6 +580,7 @@ private void aecpgenkill(ref GlobalOptimizer go, int flowxx)
                     n.EV.E1.Eoper == OPvar &&
                     n.EV.E2.Eoper == OPvar &&
                     !((n.EV.E1.Ety | n.EV.E2.Ety) & (mTYvolatile | mTYshared)) &&
+                    !((n.EV.E1.EV.Vsym.Sflags | n.EV.E2.EV.Vsym.Sflags) & SFLlivexit) &&
                     n.EV.E1.EV.Vsym != n.EV.E2.EV.Vsym)
                 {
                     n.Nflags |= NFLaecp;
```

Comment:
so NFLaecp stands for NFlags.AssignExp CopyPropagration?

Reply:
That was removed by un-merged PR https://github.com/dlang/dmd/pull/11098